app/views/handlers/confirmation_email.py

```diff
@@ -0,0 +1,31 @@
+from flask_babel import gettext
+
+from app.forms.email_form import EmailForm
+from app.globals import get_session_store
+from app.helpers.url_param_serializer import URLParamSerializer
+from app.views.contexts.email_form_context import build_confirmation_email_form_context
+
+
+class ConfirmationEmail:
+    DEFAULT_PAGE_TITLE = "Confirmation email"
```

Comment:
- I think calling it `DEFAULT_PAGE_TITLE` is confusing as to me it implies it's a fallback title. I think just calling it `PAGE_TITLE` would be clearer.
- This needs to be wrapped in a call to gettext so it is translated.
```suggestion
    PAGE_TITLE = gettext("Confirmation email")
```